### PR TITLE
Update run_dnabert2.sh

### PR DESCRIPTION
**Pull request by Sudhendra**
There is a fundamental issue with this bash script.

1. The learning rate is undefined which produces error for every run. I have provided a standard LR used for BERT (2e-5). 

Requesting the authors of DNABERT-2 to specify the LR used for finetuning on each dataset of GUE dataset in order to reproduce the results. That would be extremely helpful.

2. I have also added script for handling any incoming errors while training which specifically points out where (in which training loop of the finetuning datasets) did the error occur. Which will be very helpful for debugging later on.

### CODE REVIEW

**Comment by Sudhendra** [comment]
Needless to say, the proposed changes in the pull requests effectively resolve the problems with the DNABERT-2 codebase & I hope these proposed changes will encourage further research in the area. I will be looking ahead to polish the code further if necessary.
